barretenberg/cpp/src/barretenberg/sumcheck/sumcheck.hpp

```diff
@@ -65,7 +65,6 @@ template <typename Flavor> class SumcheckProver {
         , round(multivariate_n)
         , partially_evaluated_polynomials(multivariate_n){};
 
-    // WORKTODO delete this
```

Comment:
This was just hanging around. I tried to delete the function below and I couldn't. Must have thought it was a temporary thing but decided to keep it.